icons/banknote-arrow-up.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "AnnaSasDev",
+    "joffx"
```

The original contributors of `banknote` should also have been added, since it's the icon this was based upon:

```suggestion
    "AnnaSasDev",
    "joffx",
    "ericfennis",
    "danielbayley"
```